server/chat-plugins/repeats.ts

```diff
@@ -146,28 +159,76 @@ export const commands: ChatCommands = {
 		this.runBroadcast();
 		this.sendReplyBox(
 			`<code>/repeat [minutes], [phrase]</code>: repeats a given phrase every [minutes] minutes.<br />` +
+			`<code>/repeatfaq [minutes], [FAQ name/alias]</code>: repeats a given Room FAQ every [minutes] minutes.<br />` +
 			`<code>/removerepeat [phrase]</code>: removes a repeated phrase.<br />` +
```

Comment:
This should probably mention that you can specify a room in PMs.

Reply:
I intentionally made the split message very odd so that people wouldnt do it for other rooms in PMs/rooms

Reply:
Ah, OK, that's fair.

Reply:
Actually, I'll do the name thing now